### PR TITLE
Updated by me [jagan]

### PR DESCRIPTION
**Pull request by Jagan-1995**
As Hi priya

### CODE REVIEW

File: about-me.txt [hunk 1]
```diff
@@ -8,3 +8,5 @@ i am 22.
 i will be a good person.
 
 i can do it.
+
+Hi priya
```
Comment (Priyaaaaanu) [inline about-me.txt:12]: please say why did you add this line.

**Review by Priyaaaaanu — COMMENTED** [review]
please see the comment on line 10.

File: about-me.txt (hunk 1 above)
Comment (Jagan-1995) [reply about-me.txt:12]: it was line no 12

File: about-me.txt [hunk 2]
```diff
@@ -8,3 +8,5 @@ i am 22.
 i will be a good person.
 
 i can do it.
```
Comment (Priyaaaaanu) [inline about-me.txt:10]: any changes done.

**Review by Priyaaaaanu — APPROVED** [review]
yeah, u can improve by taking some time.
